tool/lib/core_assertions.rb

```diff
@@ -825,7 +825,7 @@ def assert_linear_performance(seq, rehearsal: nil, pre: ->(n) {n})
         end
         times.compact!
         tmin, tmax = times.minmax
-        tbase = 10 ** Math.log10(tmax * ([(tmax / tmin), 2].max ** 2)).ceil
+        tbase = 10 * tmax * [(tmax / tmin) ** 2 / 4, 1].max
```

Comment:
Could you add a comment why we use this calculation?

Reply:
Done :+1: